pinot-core/src/main/java/org/apache/pinot/core/query/request/context/ThreadTimer.java

```diff
@@ -55,7 +55,7 @@ public void stop() {
   }
 
   public long getThreadTimeNs() {
-    if (_startTimeNs == -1 || _endTimeNs == -1) {
+    if (!IS_THREAD_CPU_TIME_MEASUREMENT_ENABLED) {
```

Comment:
This check is actually redundant

Reply:
You are right, if thread cpu timing is not enabled, `_startTimeNs == -1 && _endTimeNs == -1`, so `_endTimeNs - _startTimeNs = 0`

Reply:
done.